impl/src/main/resources/org/jboss/cdi/tck/tests/full/context/passivating/passivating/enterprise/valid/ejb-jar.xml

```diff
@@ -8,8 +8,8 @@
 
       <session>
          <ejb-name>Elephant</ejb-name>
-         <business-local>org.jboss.cdi.tck.tests.context.passivating.enterprise.valid.ElephantLocal</business-local>
-         <ejb-class>org.jboss.cdi.tck.tests.context.passivating.enterprise.valid.Elephant</ejb-class>
+         <business-local>org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid.ElephantLocal</business-local>
+         <ejb-class>org.jboss.cdi.tck.tests.full.context.passivating.passivating.enterprise.valid.Elephant</ejb-class>
```

Comment:
Still uses duplicate `passivating`.

Reply:
This needs to be moved into the web profile version of the tests as the core tests should have no dependencies on other containers.